www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.php

```diff
@@ -107,6 +108,52 @@ private function checkSubdomainsAgainstDomains($subdomains, $domains, $messages)
         }
     }
 
+    // Get the OPNsense WebGUI configuration and store it
+    private $webgui;
+
+    private function getWebGui() {
+        if (!$this->webgui) {
+            $this->webgui = Config::getInstance()->object()->system->webgui ?? null;
+        }
+        return $this->webgui;
+    }
+
+    // Get the default OPNsense WebGUI ports
+    private function getWebGuiPorts() {
+        $webgui = $this->getWebGui();
+        $webGuiPorts = [];
+
+        // Only add ports to array if no specific interfaces for the WebGUI are set
+        if (empty($webgui->interfaces)) {
```

Comment:
if we assume `$webgui` could be empty, this should be `!empty($webgui) && empty((string)$webgui->interfaces)`, the cast to string ensures we're not comparing the object, which is likely safer.